bigtop-tests/smoke-tests/qfs/build.gradle

```diff
@@ -0,0 +1,40 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p/>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p/>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+def tests_to_include() {
+  return [
+      "TestHadoopExamples.groovy"
+  ];
+}
+
+sourceSets {
+  test {
+    groovy {
+      srcDirs = [
+        "${BIGTOP_HOME}/bigtop-tests/test-artifacts/hadoop/src/main/groovy/org/apache/bigtop/itest/hadoop/mapreduce"
+      ]
+      exclude {
+        FileTreeElement elem -> (doExclude(elem.getName()))
+      }
+    }
+  }
+}
+
+test.doFirst {
+  environment("HADOOP_COMMAND", "hadoop-qfs")
```

Comment:
This isn't what I meant. Someone can set the SMOKE_TESTS variable to qfs as he is intent to run such tests. However, the cluster might not be configured with qfs, i.e. the packages aren't installed, etc. This will lead to a false negative. The reason I am being overly cautious is because in case of HDFS it is always there, so there's no need to specifically check for its presence. QFS being an optional alternative won't be implicitly required by other downstream components like YARN, hence it configuration might be omitted. See where I am going with this?